lib/dpl/provider/elastic_beanstalk.rb

```diff
@@ -68,7 +68,8 @@ def version_label
       end
 
       def version_description
-        context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description = context.env['ELASTIC_BEANSTALK_DESCRIPTION'] || commit_msg
+        description.gsub(/^[\u0009\u000A\u000D\u0020-\uD7FF\uE000-\uFFFD\u10000-\u10FFFF]/, "").strip
```

Comment:
Check whether we stripped any character, and then display the message to this effect, so that the user knows that this happened.